jira_agile_metrics/calculators/progressreport.py

```diff
@@ -506,13 +503,13 @@ def update_story_counts(
     done_column
 ):
     backlog_column_index = [s['name'] for s in cycle].index(backlog_column)
-    started_column = cycle[backlog_column_index + 1]['name']  # config parser ensures there is at least one column after backlog
-    
+    committed_column = cycle[backlog_column_index + 1]['name']  # config parser ensures that `backlog` comes immediately before `committed`
```

Comment:
... or maybe I made that assumption. :) Is it right?

Reply:
As described in https://github.com/DeloitteDigitalUK/jira-agile-metrics/pull/27#issuecomment-636308642 i think that the timer for cycle time starts ticking when an issue enters `Committed column`.  It was never explicitly stated that the `Backlog column` should immediately precede the `Committed column`. That's the reason I added an explicit check in `config.py`.